test/integration/suspend-resume.spec.js

```diff
@@ -1,14 +1,11 @@
+import uuid from 'uuid/v4';
 import WebSocket from 'ws';
 import Qix from '../../src/qix';
 import Schema from '../../schemas/12.20.0.json';
 
-function generateId() {
-  return Math.floor(Math.random() * 10000).toString();
-}
-
 function buildUrl(ttl) {
   ttl = typeof ttl !== 'undefined' ? ttl : 3600;
-  return `ws://localhost:9076/app/engineData/ttl/${ttl}/identity/${generateId()}`;
+  return `ws://localhost:9076/app/engineData/ttl/${ttl}/identity/${uuid()}`;
```

Comment:
Instead of introducing a new (dev) dependency for such a small thing, perhaps use the built-in crypto lib instead? https://nodejs.org/api/crypto.html#crypto_crypto_randombytes_size_callback

Reply:
Good idea, will do!